velox/exec/tests/TaskTest.cpp

```diff
@@ -26,6 +28,11 @@ using namespace facebook::velox;
 namespace facebook::velox::exec::test {
 class TaskTest : public HiveConnectorTestBase {
  protected:
+  static void SetUpTestCase() {
+    facebook::velox::exec::ExchangeSource::registerFactory(
```

Comment:
There is a LocalExchangeSource that can be used to testing.  It is defined in velox/exec/Exchange.cpp and currently used in velox/exec/tests/MultiFragmentTest.cpp

Reply:
Thanks for this suggestion!
In this unit test, we are trying to track whether the downstream has properly closed the right exchangeSource upon its abortion. If we were supposed to use the existing `LocalExchangeSource`, we might need to add additional variables and methods to `LocalExchangeSource` to track such information, which, in my opinion, looks messier than defining a new mock ExchangeSource for our use case.